src/ram/src/MakeRam.cpp

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024-2025, The OpenROAD Authors
+
+#include "ram/MakeRam.h"
+
+#include <tcl.h>
+
+#include "ord/OpenRoad.hh"
+#include "ram/ram.h"
+#include "sta/StaMain.hh"
+
+namespace sta {
+// Tcl files encoded into strings.
+extern const char* ram_tcl_inits[];
+}  // namespace sta
+
+extern "C" {
+extern int Ram_Init(Tcl_Interp* interp);
+}
+
+namespace ord {
+
+ram::RamGen* makeRamGen()
+{
+  return new ram::RamGen;
+}
+
+void deleteRamGen(ram::RamGen* ram_gen)
+{
+  delete ram_gen;
+}
+
+void initRamGen(OpenRoad* openroad)
```

Comment:
This function needs to be adjusted so that it matches the declaration after the merge. You can look at any other tool to see how to use the new style of initialization.

This is what's currently causing the build error.